Makefile

```diff
@@ -0,0 +1,12 @@
+# Origami Service Makefile
+# ------------------------
+# This section of the Makefile should not be modified, it includes
+# commands from the Origami service Makefile.
+# https://github.com/Financial-Times/origami-service-makefile
+include node_modules/@financial-times/origami-service-makefile/index.mk
+# [edit below this line]
+# ------------------------
+
+npm-publish:
+	npx -p npm-prepublish@^1 npm-prepublish --verbose
```

Comment:
Circle doesn't seem to like this.  Do you need to install npx somewhere?

Reply:
Fixed by installing npm version 5 👍